packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -0,0 +1,174 @@
+                    {label}
+                  </h5>
+                )}
+
+                <ul
+                  aria-labelledby={label ? this.labelId : null}
+                  className="ds-c-list--bare"
+                  role="listbox"
+                >
+                  {items
+                    .filter(
+                      item =>
+                        !inputValue ||
+                        item.name
+                          .toLowerCase()
+                          .includes(inputValue.toLowerCase())
+                    )
+                    .map((item, index) => (
+                      <li
+                        aria-selected={highlightedIndex === index}
+                        className={
+                          highlightedIndex === index
+                            ? 'ds-c-autocomplete__list-item ds-c-autocomplete__list-item--active'
+                            : 'ds-c-autocomplete__list-item'
+                        }
+                        key={item.id}
+                        role="option"
+                        {...getItemProps({ item })}
+                      >
+                        {itemToString(item)}
+                      </li>
+                    ))}
+                </ul>
+              </div>
+            ) : null}
+
+            <Button
+              aria-label={clearAriaLabel}
+              className="ds-u-float--right ds-u-padding-right--0"
+              href="javascript:void(0);"
+              onClick={clearSelection}
+              size="small"
+              variation="transparent"
+            >
+              {clearInputText}
+            </Button>
+          </div>
+        )}
+      />
+    );
+  }
+}
+
+Autocomplete.defaultProps = {
+  clearAriaLabel: 'Clear typeahead and search again',
+  clearInputText: 'Clear search'
+};
+
+Autocomplete.propTypes = {
+  children: PropTypes.node,
+  /**
+   * Screenreader-specific label for the Clear input link. Intended to provide a longer, more descriptive explanation of the link's behavior.
+   */
+  clearAriaLabel: PropTypes.string,
+  /**
+   * Clear link text that will appear on the page as part of the rendered component
+   */
+  clearInputText: PropTypes.node,
+  /**
+   * Used to determine the string value for the selected item (which is used to compute the `inputValue`.
+   *
+   * Also see: https://github.com/paypal/downshift#itemtostring
+   */
+  itemToString: PropTypes.func,
+  /**
+   * Array of objects used to populate the suggestion list that appears below the input as users type. This array of objects is intended for an async data callback, and should conform to the prescribed shape to avoid errors.
+   */
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      name: PropTypes.name
+    })
+  ).isRequired,
+  /**
+   * Adds a heading to the top of the autocomplete list. This is used to convey to the user that they're required to select an option from the autocomplete list.
```

Comment:
✍️  Content suggestion:

Change "This **is** used..." to "This **can be** used", since this is just one way the `label` prop can be used now.

Reply:
Done. Changed to can be used... Reads better now.